examples/get_started/02_contour_map.py

```diff
@@ -75,35 +75,38 @@
 # Adding contour lines
 # --------------------
 #
-# To add contour lines to the color coded figured, the
+# To add contour lines to the color-coded figure, the
 # :meth:`pygmt.Figure.grdcontour` method is used. The ``frame`` and
 # ``projection`` are already set using :meth:`pygmt.Figure.grdimage` and are
 # not needed again. However, the same input for ``grid`` (in this case, the
 # variable named "grid") must be input again. The ``interval`` parameter sets
-# the spacing between lines (in this case, 500 meters), and the ``annotation``
-# parameter draws darker lines that are annotated with the elevation or
-# bathymetry.
+# the spacing between adjacent contour lines (in this case, 500 meters). The
+# ``annotation`` parameter annotates the contour lines corresponding to the
+# given interval (in this case, 1,000 meters) with the related values, here
+# elevation or bathymetry. By default, these contour lines are drawn thicker.
+# The appearance (thickness, color, style) of the contour lines can be adjusted
+# by specifying a desired ``pen``, which then applies to all contour lines.
```

Comment:
Do we want to keep this part, or should I remove it, as the usage of `pen` is not explicitly shown in this tutorial?

Reply:
Would leave it and add that it's optional.

Reply:
Thanks @michaelgrund. I like this solution and applied your formulation suggestion.